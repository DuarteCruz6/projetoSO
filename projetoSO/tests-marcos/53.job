WRITE [(h9,48)(b3,46)(b9,10)]
DELETE [b9]
READ [h9]
WAIT 2000
DELETE [b3]
DELETE [h9]
WAIT 2000
WRITE [(d0,74)(j0,44)(o4,31)(b1,88)(x8,54)]
READ [d0]
WRITE [(g2,20)(c4,51)(q9,15)(t5,100)(s3,64)]
WRITE [(m0,81)(m5,56)]
READ [b1]
WRITE [(e4,59)(e4,55)(z9,89)(f5,89)]
WRITE [(l8,86)(n0,64)(z0,28)(s4,95)(r0,96)]
READ [r0]
WAIT 3000
WRITE [(a2,8)(t3,28)(g8,70)(o9,82)(u8,19)]
SHOW
SHOW
DELETE [f5,g2,z9,j0]
WRITE [(t6,22)]
WRITE [(g1,90)(g4,8)(i2,30)]
WRITE [(j5,56)(d1,8)]
WRITE [(r2,6)(z3,93)]
WAIT 1000
BACKUP
SHOW
WRITE [(o6,82)(a3,90)(n7,92)(f3,79)]
WAIT 2000
DELETE [o9]
WAIT 1000
WRITE [(t2,36)(c2,43)(v1,5)(l1,30)(t7,42)]
WRITE [(m8,81)(l0,97)(l3,29)(z5,6)(j8,30)]
WRITE [(f6,62)]
READ [s4]
READ [r2]
SHOW
WRITE [(o2,17)(e1,65)]
WRITE [(b6,52)(f4,28)(s2,10)(s8,61)(z9,28)]
READ [e4]
WRITE [(u2,63)(d2,56)]
WRITE [(x4,60)(k3,4)(t0,41)(q1,78)(b4,1)]
WRITE [(d8,52)(k1,6)(g3,46)(e0,76)]
WAIT 3000
WRITE [(z7,31)(x6,80)(u7,48)(m2,2)(b3,44)]
WRITE [(j9,72)(z6,21)(m4,57)]
WRITE [(f8,66)(x5,51)(a4,80)(w2,3)]
DELETE [j8,s8]
WRITE [(g2,29)(p6,66)(h0,65)(k8,6)(v3,31)]
WRITE [(x3,34)(o5,33)(n8,67)]
BACKUP
WRITE [(y5,67)(z2,49)(c6,19)(n1,15)(v4,34)]
WRITE [(g6,2)(w0,82)(i0,93)(q4,59)(w7,62)]
READ [t6]
WRITE [(w4,85)]
SHOW
WRITE [(k9,29)(n3,26)]
WRITE [(t9,19)(h3,7)(o9,31)(w9,8)]
WRITE [(p1,82)(c9,43)]
WRITE [(b2,44)(n9,54)]
WRITE [(l6,73)(t4,84)]
READ [j5]
WAIT 2000
WRITE [(m6,64)(g9,38)(y8,2)(b7,67)]
WAIT 2000
WRITE [(l2,14)(x0,23)(w8,92)(i8,21)(p9,83)]
WRITE [(j8,99)]
WRITE [(s1,68)]
READ [x8]
WRITE [(e3,19)(d7,77)(x9,10)(n4,71)]
DELETE [d1]
WRITE [(v8,10)]
READ [k1]
WRITE [(h5,39)(w3,74)(f0,69)(i9,0)(b9,74)]
WRITE [(j1,6)(k6,24)(n6,66)(y7,78)(a5,96)]
READ [t3]
READ [u2]
WRITE [(z4,28)(w5,72)]
WRITE [(v6,83)(h4,100)(k0,29)]
WRITE [(e8,23)(s8,54)(n2,90)]
WRITE [(q0,84)(k2,63)(d6,9)]
WRITE [(i5,94)(a6,10)]
WRITE [(a0,62)(p2,72)(k4,78)(a9,69)]
WRITE [(u3,97)(j0,2)(j3,95)(b8,97)]
WRITE [(e5,85)(e7,5)]
WRITE [(j2,73)(v9,67)]
DELETE [f0]
WRITE [(d1,26)]
WRITE [(y3,73)]
READ [i8]
WRITE [(f9,3)(d9,56)(i7,71)(p8,22)]
WRITE [(p4,85)(d3,85)(m1,77)(o8,52)]
READ [d0]
WRITE [(f7,80)]
WRITE [(y4,57)(d4,29)(v0,50)]
