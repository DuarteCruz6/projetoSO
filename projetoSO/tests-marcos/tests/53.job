WRITE [(e7,37)(s5,80)(w0,92)]
WRITE [(g4,99)]
READ [e7]
WAIT 3000
WRITE [(n5,77)]
WRITE [(w9,5)(a6,83)(d7,49)]
READ [n5]
WRITE [(b5,53)(e2,76)]
DELETE [s5,b5,e2,w9]
WRITE [(u3,74)(r3,51)(j9,56)(p5,33)(h2,60)]
WRITE [(w3,15)]
WRITE [(f5,35)(j1,26)]
WRITE [(t2,4)(p8,54)(v1,33)(p9,15)(r1,36)]
WRITE [(w9,0)(d4,12)(s5,61)(c6,56)(i9,5)]
READ [j9]
WRITE [(s0,95)(b8,38)(v9,100)(g1,64)(t8,92)]
READ [t2]
WRITE [(u4,23)(x8,91)(q3,86)(y6,24)(r9,52)]
WRITE [(l7,42)(m8,73)(j0,58)]
WRITE [(k1,17)(t6,89)(i8,12)(p4,66)]
WAIT 2000
READ [u3]
WRITE [(j8,33)(t7,51)]
WRITE [(a7,13)]
READ [r3]
WRITE [(s8,75)(y0,57)(y9,81)]
WRITE [(q5,95)(r0,44)(l4,55)(p2,75)(z5,67)]
WRITE [(q2,47)(i7,39)(u8,39)(w8,69)]
DELETE [w0,y0,n5]
WRITE [(m4,65)(b0,28)(w2,58)(r4,15)]
WRITE [(z1,57)(y1,52)(k5,35)(k8,6)(p7,36)]
WRITE [(y3,15)(n9,4)(x6,17)(y8,14)]
DELETE [m8,t7,c6,l7,p5]
READ [a6]
WRITE [(a9,27)(j4,89)(j6,9)]
DELETE [s8,n9,k8]
DELETE [u3,d7,w8,j1,j8]
WAIT 1000
READ [j6]
READ [w9]
WRITE [(i6,49)(r6,24)(e8,79)(m7,89)]
READ [q2]
READ [w3]
READ [v1]
READ [y1]
READ [l4]
WRITE [(w5,93)(q8,64)(l6,4)]
READ [a6]
DELETE [q5]
READ [g4]
BACKUP
WRITE [(m9,35)(s4,38)(b4,86)]
WRITE [(m3,32)(h0,52)(w8,38)(v0,67)]
WRITE [(u2,58)(l5,63)(a5,55)(w1,9)(k7,37)]
WRITE [(n2,63)]
SHOW
WRITE [(a4,1)]
WRITE [(a2,39)(v4,13)(j5,93)(u1,14)]
READ [g1]
WRITE [(n5,20)]
READ [j4]
WRITE [(w7,4)]
SHOW
WRITE [(c9,26)]
WRITE [(m8,54)]
WRITE [(x4,21)]
WRITE [(n7,81)(m2,35)(z2,54)(y4,76)(c3,45)]
WRITE [(h9,1)(p0,65)(d2,86)(t9,22)(g6,37)]
WRITE [(o1,31)(o4,59)(b2,68)]
WRITE [(x3,0)(g0,12)]
WRITE [(e6,98)(p5,38)(v2,80)(j7,74)]
DELETE [k5,q3,i7,u2,e6]
WRITE [(d8,83)(k9,31)]
WRITE [(s1,20)(v8,31)(b1,32)(l1,67)(x9,73)]
DELETE [l1,a9,o1]
WRITE [(u7,95)(r2,76)(l2,97)(h7,2)(u3,9)]
WRITE [(n0,23)]
WRITE [(c1,91)(l1,70)]
READ [k9]
WRITE [(f1,86)(b6,13)(g5,75)(k8,18)(x2,19)]
DELETE [h2,a7]
READ [u1]
READ [n2]
READ [p0]
READ [b8]
WRITE [(n1,46)(x1,29)(d9,38)(t0,61)(g3,75)]
WRITE [(q6,73)(j8,15)(t3,20)(x5,4)(k5,95)]
READ [r9]
WRITE [(o2,68)(e3,9)(g8,50)(x0,89)(q4,25)]
READ [v8]
WRITE [(s9,2)(m0,22)(b7,71)]
READ [x1]
WRITE [(m1,7)]
WRITE [(l8,9)]
WRITE [(b5,25)(e5,22)(e4,91)(q0,15)]
WRITE [(r7,59)(w4,22)(v7,52)]
WRITE [(v5,68)(i1,82)(o9,56)]
