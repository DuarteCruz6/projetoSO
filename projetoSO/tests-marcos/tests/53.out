[(e7,37)]
[(n5,77)]
[(j9,56)]
[(t2,4)]
[(u3,74)]
[(r3,51)]
[(a6,83)]
[(j6,9)]
[(w9,0)]
[(q2,47)]
[(w3,15)]
[(v1,33)]
[(y1,52)]
[(l4,55)]
[(a6,83)]
[(g4,99)]
(a5, 55)
(a9, 27)
(a7, 13)
(a6, 83)
(b4, 86)
(b0, 28)
(b8, 38)
(d4, 12)
(e8, 79)
(e7, 37)
(f5, 35)
(g1, 64)
(g4, 99)
(h0, 52)
(h2, 60)
(i6, 49)
(i7, 39)
(i8, 12)
(i9, 5)
(j6, 9)
(j4, 89)
(j0, 58)
(j9, 56)
(k7, 37)
(k5, 35)
(k1, 17)
(l5, 63)
(l6, 4)
(l4, 55)
(m3, 32)
(m9, 35)
(m7, 89)
(m4, 65)
(n2, 63)
(p7, 36)
(p2, 75)
(p4, 66)
(p9, 15)
(p8, 54)
(q8, 64)
(q2, 47)
(q3, 86)
(r6, 24)
(r4, 15)
(r0, 44)
(r9, 52)
(r1, 36)
(r3, 51)
(s4, 38)
(s0, 95)
(s5, 61)
(t6, 89)
(t8, 92)
(t2, 4)
(u2, 58)
(u8, 39)
(u4, 23)
(v0, 67)
(v9, 100)
(v1, 33)
(w1, 9)
(w8, 38)
(w5, 93)
(w2, 58)
(w9, 0)
(w3, 15)
(x6, 17)
(x8, 91)
(y8, 14)
(y3, 15)
(y1, 52)
(y9, 81)
(y6, 24)
(z1, 57)
(z5, 67)
[(g1,64)]
[(j4,89)]
(a2, 39)
(a4, 1)
(a5, 55)
(a9, 27)
(a7, 13)
(a6, 83)
(b4, 86)
(b0, 28)
(b8, 38)
(d4, 12)
(e8, 79)
(e7, 37)
(f5, 35)
(g1, 64)
(g4, 99)
(h0, 52)
(h2, 60)
(i6, 49)
(i7, 39)
(i8, 12)
(i9, 5)
(j5, 93)
(j6, 9)
(j4, 89)
(j0, 58)
(j9, 56)
(k7, 37)
(k5, 35)
(k1, 17)
(l5, 63)
(l6, 4)
(l4, 55)
(m3, 32)
(m9, 35)
(m7, 89)
(m4, 65)
(n5, 20)
(n2, 63)
(p7, 36)
(p2, 75)
(p4, 66)
(p9, 15)
(p8, 54)
(q8, 64)
(q2, 47)
(q3, 86)
(r6, 24)
(r4, 15)
(r0, 44)
(r9, 52)
(r1, 36)
(r3, 51)
(s4, 38)
(s0, 95)
(s5, 61)
(t6, 89)
(t8, 92)
(t2, 4)
(u1, 14)
(u2, 58)
(u8, 39)
(u4, 23)
(v4, 13)
(v0, 67)
(v9, 100)
(v1, 33)
(w7, 4)
(w1, 9)
(w8, 38)
(w5, 93)
(w2, 58)
(w9, 0)
(w3, 15)
(x6, 17)
(x8, 91)
(y8, 14)
(y3, 15)
(y1, 52)
(y9, 81)
(y6, 24)
(z1, 57)
(z5, 67)
[(k9,31)]
[(u1,14)]
[(n2,63)]
[(p0,65)]
[(b8,38)]
[(r9,52)]
[(v8,31)]
[(x1,29)]
